src/pathfinder/yapf/yapf_base.hpp

```diff
@@ -56,10 +59,10 @@ class CYapfBaseT {
 
 	NodeList             m_nodes;              ///< node list multi-container
 protected:
+	static const int MAX_SEARCH_NODES = 10000; ///< stop path-finding when this number of nodes visited
```

Comment:
Seems odd to stick this constant here when the others are defined at the top.

Reply:
I forgot to move the ship constants :)

But this constant is here (inside the class) in a vague attempt to hide it from the global scope where it's not needed (the other constants (except rail) are in cpp files)